### ISSUE TITLE
Error when pressing Ctrl+C in the terminal where Spyder was started 

### ISSUE THREAD

**Issue by by965738071** [spyder-ide/spyder#5305]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/home/salalaika/application/Anaconda3/lib/python3.6/site-packages/spyder/widgets/status.py", line 108, in update_label
    def update_label(self):
KeyboardInterrupt

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.1
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.5.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.2.2 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.25.2 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  5.3.0 (OK)
pylint >=0.25    :  1.6.4 (OK)

```

**Comment by ccordoba12** [comment]
This will be fixed in Spyder 3.2.4. Thanks for reporting.

**Comment by rlaverde** [comment]
@ccordoba12 It think It's related to #5114, although It isn't a real error, pressing Ctr+C in the terminal running spyder will popup the error dialog.

Maybe #5159 isn't the best approach but we should avoid these false positive errors reports

**Comment by ccordoba12** [comment]
Yep, I know. That's why I said it'll be fixed in 3.2.4 :-)

I just want to give you a bit of feedback on that PR and then I'll merge it.
